CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 15.5.4 (April 11, 2017)
+
+### React Test Renderer
+
+* Fix compatibility with Enzyme by exposing `batchedUpdates` on shallow renderer. [See #9382](https://github.com/facebook/react/commit/69933e25c37cf5453a9ef132177241203ee8d2fd).
+
+
+## 15.5.3 (April 7, 2017)
+
+### React Addons
+
+* Fix react-addons-create-fragment package to export correct thing. ([@gaearon](https://github.com/gaearon) in [#9385](https://github.com/facebook/react/pull/9383))
```

Comment:
Nit: lets use backticks for package names. We don't use them for proper names though (e.g. we either write React DOM or `react-dom`)